content/blue_green_deployments/tabs/code-push.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Embedded tab content"
+disableToc: true
+hidden: true
+---
+
+#### Push the source code to the CodeCommit repository
+The stack created an empty CodeCommit repository - `demo-app`. Now we will populate it with a sample demo application.
+
+#### Clone the demo application
+```bash
+cd ~/environment
+git clone https://github.com/smuralee/nginx-example.git
+```
+
+#### Update the remote origin URL
+Change the remote origin to the CodeCommit repository. You can get the repository URL from the output value of `BlueGreenUsingEcsStack.ecsBlueGreenCodeRepo` in the stack output. Here I am assuming the region is `us-east-1`
+
+```bash
+cd ~/environment/nginx-example
+git remote set-url origin https://git-codecommit.us-east-1.amazonaws.com/v1/repos/demo-app
```

Comment:
When I ran the git push, it didn’t work. Please dynamically set the git url. 

Something like this:
```bash
git remote set-url origin $(aws-codecommit get-repository —repository-name demo-app | jq -r repositoryMetadata.cloneUrlHttp)
```

Reply:
Added dynamic setting of the origin url using AWS CLI